src/Raven.Studio/typescript/models/database/status/storageReportItem.ts

```diff
@@ -19,15 +20,18 @@ class storageReportItem {
     numberOfEntries: number = null;
     lazyLoadChildren = false;
     customSizeProvider: (header: boolean) => string;
-
+    isGrouped: boolean;
+    
     recyclableJournal = false;
 
-    constructor(name: string, type: string, showType: boolean, size: number, internalChildren: storageReportItem[] = null) {
+    constructor(name: string, type: string, showType: boolean, size: number, length: number, internalChildren: storageReportItem[] = null, isGrouped: boolean = false) {
```

Comment:
if we need length param but it is null 90% of use cases then don't put that as constructor param - set this using property - only when needed.

Reply:
done